src/Compilers/CSharp/Test/Semantic/Semantics/SwitchTests.cs

```diff
@@ -2949,6 +2949,42 @@ public static void Main()
                 Diagnostic(ErrorCode.WRN_UnreachableCode, "break").WithLocation(14, 17));
         }
 
+        [Fact, WorkItem(47878, "https://github.com/dotnet/roslyn/issues/47878")]
+        public void Bug47878()
+        {
+            var text = @"
+using System;
+public class C
+{   
+    public static void Main()
+    {
+        int x1 = (1, 1, 1, 1, 1, 1, 1, 1, 1, 1) switch
```

Comment:
Please split out this test and verify the IL. I don't think we need to verify IL on all of these variations, but it would be good to have 1 of them verified.

Reply:
@333fred I feel it's safer to verify the IL for different cases because in the past, the compiler had different behavior between the 8 elements case, and the > 8 elements case.

I'd at least verify the IL for 2 cases:

- 8 elements
- Greater than 8 elements